test/CadlRanchProjects/encode/bytes/src/Generated/Models/Base64urlArrayBytesProperty.cs

```diff
@@ -35,29 +35,15 @@ internal Base64urlArrayBytesProperty(IList<BinaryData> value)
         /// <summary>
         /// Gets the value
         /// <para>
-        /// To assign an object to the element of this property use <see cref="BinaryData.FromObjectAsJson{T}(T, System.Text.Json.JsonSerializerOptions?)"/>.
-        /// </para>
-        /// <para>
-        /// To assign an already formated json string to this property use <see cref="BinaryData.FromString(string)"/>.
+        /// To assign a byte[] to the element of this property use <see cref="BinaryData.FromBytes(byte[])"/>.
+        /// The byte[] will be serialized to a Base64 encoded string.
         /// </para>
         /// <para>
         /// Examples:
         /// <list type="bullet">
         /// <item>
-        /// <term>BinaryData.FromObjectAsJson("foo")</term>
-        /// <description>Creates a payload of "foo".</description>
-        /// </item>
-        /// <item>
-        /// <term>BinaryData.FromString("\"foo\"")</term>
-        /// <description>Creates a payload of "foo".</description>
-        /// </item>
-        /// <item>
-        /// <term>BinaryData.FromObjectAsJson(new { key = "value" })</term>
-        /// <description>Creates a payload of { "key": "value" }.</description>
-        /// </item>
-        /// <item>
-        /// <term>BinaryData.FromString("{\"key\": \"value\"}")</term>
-        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// <term>BinaryData.FromBytes(new byte[] { 1, 2, 3 })</term>
+        /// <description>Creates a payload of "AQID".</description>
```

Comment:
This change is expected.
The item of this list has an encode of base 64 string. Therefore our help message should write this information just like we did in the normal property here: https://github.com/ArcturusZhang/autorest.csharp/blob/2ff52249838b64f59584cbfa07e9ac5563522429/test/CadlRanchProjects/encode/bytes/src/Generated/Models/Base64BytesProperty.cs#L29